{"name": "Class TransformerException", "module": "java.xml", "package": "javax.xml.transform", "text": "This class specifies an exceptional condition that occurred\n during the transformation process.", "codes": ["public class TransformerException\nextends Exception"], "fields": [], "methods": [{"method_name": "getLocator", "method_sig": "public SourceLocator getLocator()", "description": "Method getLocator retrieves an instance of a SourceLocator\n object that specifies where an error occurred."}, {"method_name": "setLocator", "method_sig": "public void setLocator (SourceLocator location)", "description": "Method setLocator sets an instance of a SourceLocator\n object that specifies where an error occurred."}, {"method_name": "getException", "method_sig": "public Throwable getException()", "description": "This method retrieves an exception that this exception wraps."}, {"method_name": "getCause", "method_sig": "public Throwable getCause()", "description": "Returns the cause of this throwable or null if the\n cause is nonexistent or unknown.  (The cause is the throwable that\n caused this throwable to get thrown.)"}, {"method_name": "initCause", "method_sig": "public Throwable initCause (Throwable cause)", "description": "Initializes the cause of this throwable to the specified value.\n (The cause is the throwable that caused this throwable to get thrown.)\n\n This method can be called at most once.  It is generally called from\n within the constructor, or immediately after creating the\n throwable.  If this throwable was created\n with TransformerException(Throwable) or\n TransformerException(String,Throwable), this method cannot be called\n even once."}, {"method_name": "getMessageAndLocation", "method_sig": "public String getMessageAndLocation()", "description": "Get the error message with location information\n appended."}, {"method_name": "getLocationAsString", "method_sig": "public String getLocationAsString()", "description": "Get the location information as a string."}, {"method_name": "printStackTrace", "method_sig": "public void printStackTrace()", "description": "Print the the trace of methods from where the error\n originated.  This will trace all nested exception\n objects, as well as this object."}, {"method_name": "printStackTrace", "method_sig": "public void printStackTrace (PrintStream s)", "description": "Print the the trace of methods from where the error\n originated.  This will trace all nested exception\n objects, as well as this object."}, {"method_name": "printStackTrace", "method_sig": "public void printStackTrace (PrintWriter s)", "description": "Print the the trace of methods from where the error\n originated.  This will trace all nested exception\n objects, as well as this object."}]}